app/api/events.py

```diff
@@ -27,6 +27,14 @@ class Meta:
                           schema='TicketSchema',
                           many=True,
                           type_='ticket')
+    notification = Relationship(attribute='notification',
+                          self_view='v1.event_notification',
+                          self_view_kwargs={'id': '<id>'},
+                          related_view='v1.notification_detail',
+                          related_view_kwargs={'id': '<id>'},
```

Comment:
I don't get where do we have to add url for NotificationDetail. Could you please explain?